pkg/fleet-manager/fleet_cluster.go

```diff
@@ -41,6 +42,9 @@ type ClusterKey struct {
 }
 
 func (f *FleetManager) buildFleetClusters(ctx context.Context, fleet *fleetapi.Fleet) (map[ClusterKey]*fleetCluster, error) {
+	log := ctrl.LoggerFrom(ctx)
+	log = log.WithValues("fleet", types.NamespacedName{Name: fleet.Name, Namespace: fleet.Namespace})
```

Comment:
We have set it from `reconcile`, @Xieql told me we donot need to set it again in any other place